.github/workflows/ci.yml

```diff
@@ -227,6 +227,13 @@ jobs:
           docker exec app bin/console glpi:database:configure --config-dir=./tests --no-interaction --reconfigure --db-name=glpi --db-host=db --db-user=root
           docker exec app vendor/bin/atoum -p 'php -d memory_limit=512M' --debug --force-terminal --use-dot-report --bootstrap-file tests/bootstrap.php --no-code-coverage --fail-if-skipped-methods --max-children-number 1 -d tests/database
           docker exec app bin/console glpi:database:configure --config-dir=./tests --no-interaction --reconfigure --db-name=glpi --db-host=db --db-user=root
+      - name: "IMAP tests"
```

Comment:
I put tests on the top, but this should be reverted before merge.

Reply:
Yes, I looking here what is the problem